python/pylibcudf/pylibcudf/tests/io/test_csv.py

```diff
@@ -161,17 +163,24 @@ def test_read_csv_dtypes(csv_table_data, source_or_sink, usecols):
 
     new_schema = pa.schema(new_fields)
 
-    res = plc.io.csv.read_csv(
-        plc.io.SourceInfo([source]), dtypes=dtypes, usecols=usecols
-    )
+    options = plc.io.csv.CsvReaderOptions.builder(
+        plc.io.SourceInfo([source])
+    ).build()
+    options.set_dtypes(dtypes)
+    if usecols is not None:
+        if all([isinstance(col, int) for col in usecols]):
```

Comment:
```suggestion
        if all(isinstance(col, int) for col in usecols):
```

Reply:
I got `closures inside cpdef functions not yet supported`. I looks like you can't use generator expressions inside cpdef functions.

Reply:
Ah OK. Didn't realize there was a cython limitation